### PR TITLE
New parser

### PR DESCRIPTION
**Pull request by melaniecebula**
Make changes to hackfile parser for new format.

Context:
https://github.com/mafintosh/hackfile/issues/2
https://github.com/datproject/gasket/issues/17

### CODE REVIEW

**Comment by mafintosh** [comment]
Small nitpick: could we use camelCase instead of snake_case? :)

**Comment by melaniecebula** [comment]
oops, I've been writing in a lot of python recently >.>

File: index.js [hunk 1]
```diff
@@ -18,10 +18,13 @@ var hackfile = function(src) {
     if (/^\s/.test(line) && !indented) throw new SyntaxError('Inconsistent indentation')
     if (!latest && indented) throw new SyntaxError('Unnamed indentation group not allowed')
 
-    if (indented) return result[latest].push(line.trim().split(/\s+/))
+    if (indented) return result[latest].push(line.trim())
+
+    latest_line = line.trim().split(/\s+/)
```
Comment (mafintosh) [inline index.js]: you probably mean `var latest_line` since otherwise this is a global variable

Comment (melaniecebula) [inline index.js]: yes, thanks!

**Comment by mafintosh** [comment]
Great work! Released as 2.0.0

**Pull request merged**